app/views/sections/items/ConfirmCommodityCodeView.scala.html

```diff
@@ -0,0 +1,43 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.requests.DataRequest
+@import viewmodels.helpers.CheckAnswersHelper
+
+@this(layout: templates.Layout,
+        govukSummaryList: GovukSummaryList,
+        formHelper: FormWithCSRF,
+        h1: components.h1,
+        saveAndContinue: components.saveAndContinue)
+
+@(submitAction: Call, summaryList: SummaryList)(implicit request: DataRequest[_], messages: Messages)
+
+    @layout(maybeShowActiveTrader = maybeShowActiveTrader(request), pageTitle = titleNoForm(messages("confirmCommodityCode.heading"))) {
+
+        @formHelper(submitAction) {
+
+            @h1("confirmCommodityCode.heading", Some(messages("items.subHeading")), "govuk-heading-l")
```

Comment:
This may fail accessibility, but I will need to check

Reply:
It's not inside a legend/label so not sure - need to double check this

Reply:
We don't have an h1 inside a legend/label on other CYA pages so I'll leave it for now, if it gets flagged in an a11y audit we'll address it with the other CYA pages